src/finch/resources/hris/benefits/benefits.py

```diff
@@ -113,9 +112,7 @@ def retrieve(
         timeout: float | httpx.Timeout | None | NotGiven = NOT_GIVEN,
     ) -> CompanyBenefit:
         """
-        **Availability: Automated Benefits providers only**
```

Comment:
@arditP is it fine that we've lost some additional context in the comments?

Reply:
I think this is fine. Our docs have this information added to the `.md` file.